### PR TITLE
BEEFMAN RUSSIAN 2

### PR DESCRIPTION
**Pull request by SmoSmoSmoSmok**
## About The Pull Request

they can SPEAK RUSSIAN 1ST PR GAVE ME MERGE CONFLICT I TRIED RESEARCHING WHAT THE FUCK A MERGE CONFLCIT IS AND TRIED TO FIX THE ISSUE TO NO AVAIL so i deleted it  and this one should work now I THINK

## Why It's Good For The Game

They can speak RUSSIAN
## Changelog

:cl:
russian speak beefemen
/:cl:

### CODE REVIEW

**Comment by SmoSmoSmoSmok** [comment]
this ONE will NOT merge conflict i swear on my mom

**Comment by StoneMason** [comment]
speedmerge

**Comment by SmoSmoSmoSmok** [comment]
> speedmerge

are U the OWNER of this wwebSite i dont have all day cna u merge this already u lazy asshole

**Comment by JohnFulpWillard** [comment]
24 hour rule

**Comment by SmoSmoSmoSmok** [comment]
> 24 hour rule

yoYOu tyrant I KNEW IT. Ur TRYING TO IMPEED THIS!!!!!!!! u PUT a MERGE CONFLICT ON MY LAST PR now ur MAKING SOME RULE UP????? ima bout to humBLE u

**Comment by JohnFulpWillard** [comment]
> > 24 hour rule
> 
> yoYOu tyrant I KNEW IT. Ur TRYING TO IMPEED THIS!!!!!!!! u PUT a MERGE CONFLICT ON MY LAST PR now ur MAKING SOME RULE UP????? ima bout to humBLE u

😐

File: fulp_modules/features/species/beefmen/mob/beefman.dm [hunk 1]
```diff
@@ -722,3 +723,34 @@
 /datum/sprite_accessory/beef/mouth/smile2
 	name = "Smile2"
 	icon_state = "smile2"
+
+/obj/item/organ/tongue/beefman_tongue
```
Comment (SgtHunk) [inline fulp_modules/features/species/beefmen/mob/beefman.dm:727]:
```suggestion
/obj/item/organ/tongue/beefman
```

File: fulp_modules/features/species/beefmen/mob/beefman.dm [hunk 2]
```diff
@@ -12,6 +12,7 @@
 	default_color = "#e73f4e"
 	species_traits = list(NOEYESPRITES, NO_UNDERWEAR, DYNCOLORS, AGENDER, HAS_FLESH, HAS_BONE)
 	mutant_bodyparts = list("beefcolor" = "Medium Rare", "beefmouth" = "Smile1", "beefeyes" = "Olives")
+	mutanttongue = /obj/item/organ/tongue/beefman_tongue
```
Comment (SgtHunk) [inline fulp_modules/features/species/beefmen/mob/beefman.dm:15]:
```suggestion
	mutanttongue = /obj/item/organ/tongue/beefman
```

File: fulp_modules/features/species/beefmen/mob/beefman.dm [hunk 3]
```diff
@@ -722,3 +723,34 @@
 /datum/sprite_accessory/beef/mouth/smile2
 	name = "Smile2"
 	icon_state = "smile2"
+
+/obj/item/organ/tongue/beefman_tongue
+	name = "meaty sausage tongue"
+	desc = "A meaty and thick muscle typically found in Beefmen."
```
Comment (SgtHunk) [inline fulp_modules/features/species/beefmen/mob/beefman.dm:729]:
```suggestion
	desc = "A meaty and thick muscle typically found in beefmen."
```

File: fulp_modules/features/species/beefmen/mob/beefman.dm [hunk 4]
```diff
@@ -722,3 +723,34 @@
...
+	icon = 'icons/obj/food/food.dmi'
+	icon_state = "raw_sausage"
```
Comment (SgtHunk) [inline fulp_modules/features/species/beefmen/mob/beefman.dm:731]: ☹️

**Review by SgtHunk — CHANGES_REQUESTED** [review]
Change the path.
Also use the sprites provided by Joyce (you'd know if you saw the beefman pr smh)

File: fulp_modules/features/species/beefmen/mob/beefman.dm (hunk 4 above)
Comment (SmoSmoSmoSmok) [reply fulp_modules/features/species/beefmen/mob/beefman.dm:731]: maaaaaaaaan where do I find the ICON

File: fulp_modules/features/species/beefmen/mob/beefman.dm (hunk 3 above)
Comment (JohnFulpWillard) [reply fulp_modules/features/species/beefmen/mob/beefman.dm]: beefmen is capitalized??

File: fulp_modules/features/species/beefmen/mob/beefman.dm (hunk 4 above)
Comment (SmoSmoSmoSmok) [reply fulp_modules/features/species/beefmen/mob/beefman.dm:731]: is it in the beefman_bodyparts.dmi ive scoured that file up and down and theres NOTHING that remotely looks like a tongue!!!!!!!

Comment (SgtHunk) [reply fulp_modules/features/species/beefmen/mob/beefman.dm:731]: https://github.com/fulpstation/fulpstation/commit/7755abdc0595bdba9747a1f9f44f60dfa2468923

Comment (SmoSmoSmoSmok) [reply fulp_modules/features/species/beefmen/mob/beefman.dm]: are we cool now

Comment (SgtHunk) [reply fulp_modules/features/species/beefmen/mob/beefman.dm]: That works

**Review by SgtHunk — APPROVED** [review]

**Pull request merged**